enterprise_catalog/apps/api/v1/export_utils.py

```diff
@@ -119,6 +142,7 @@ def course_hit_to_row(hit):
     """
     Helper function to construct a CSV row according to a single Algolia result course hit.
     """
+    exec_course = hit.get('course_type') == 'executive-education-2u' if True else False
```

Comment:
Would make the variable name more clear that it's a boolean, and also remove the extraneous logic at the end.
```suggestion
    is_exec_course = hit.get('course_type') == 'executive-education-2u'
```

Reply:
Yeah this is what we were talking about, basically its the question of do we want a lot of copied code to be cleaner and more readable, or do we want to reduce copied code. I agreed with you, but John and Alex were in the nay boat. Tbh its just preference 
https://twou.slack.com/archives/C049WNMFFRP/p1669755717967509

Reply:
I have no objections to change the variable name!